package.json

```diff
@@ -0,0 +1,86 @@
+{
+  "name": "@dsnp/activity-content",
+  "version": "1.0.0",
+  "description": "",
+  "main": "dist/index.js",
+  "types": "dist/types/index.d.ts",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/LibertyDSNP/activity-content.git"
+  },
+  "license": "ISC",
+  "scripts": {
+    "build:cjs": "tsc -p tsconfig.json",
+    "ts-types": "tsc --emitDeclarationOnly --outDir ./dist/types",
+    "build": "mkdir -p ./dist/types/ && esbuild --format=cjs ./src/index.ts src/core/index.ts --bundle --outdir=./dist --outbase=src && npm run ts-types",
+    "clean": "rm -Rf dist ./*.d.ts ./*.js ./*.tgz",
+    "doc": "rm -Rf dist/docs && npm run doc:markdown",
+    "doc:html": "typedoc --options ./typedoc.json --plugin none --theme default --out dist/docs",
+    "doc:config": "typedoc --showConfig --options ./typedoc.json --plugin none --theme default --out dist/docs",
+    "doc:json": "typedoc --options ./typedoc.json --json dist/docs/typedoc.json",
+    "doc:markdown": "typedoc --options ./typedoc.json --theme markdown --out dist/docs",
+    "format": "eslint \"**/*.{js,ts}\" --fix",
+    "lint": "eslint \"**/*.{js,ts}\"",
+    "pre-publish": "cp package.json dist",
+    "pub": "cd dist & npm publish",
+    "postpub": "cd dist & del package.json",
+    "pack": "npm run pre-publish && cd dist && npm pack && mv *.tgz ../",
```

Comment:
Publishing is including a lot more files than needed. All of these are unneeded:
```
npm notice 166B   .eslintignore
npm notice 2.2kB  .eslintrc
npm notice 67B    .github/ISSUE_TEMPLATE/config.yml
npm notice 168B   .github/ISSUE_TEMPLATE/DSNP_Discussion.md
npm notice 504B   .github/ISSUE_TEMPLATE/DSNP_Enhancement.md
npm notice 385B   .github/ISSUE_TEMPLATE/DSNP_Issue.md
npm notice 584B   .github/PULL_REQUEST_TEMPLATE.md
npm notice 336B   .idea/activity-content.iml
npm notice 8B     .node-version
npm notice 656B   jest.config.ts
npm notice 742B   src/core/errors.ts
npm notice 9.0kB  src/core/factories.test.ts
npm notice 12.4kB src/core/factories.ts
npm notice 822B   src/core/index.ts
npm notice 45.3kB src/core/validation.test.ts
npm notice 22.7kB src/core/validation.ts
npm notice 333B   src/index.ts
npm notice 581B   src/utilities/types.ts
npm notice 1.2kB  src/utilities/validation.ts
npm notice 129B   tsconfig.eslint.json
npm notice 385B   tsconfig.json
npm notice 165B   typedoc.json
```

I think the only ones we need are:

```
npm notice 728B   README.md
npm notice 43.6kB dist/core/index.js
npm notice 143B   dist/docs/.nojekyll
npm notice 480B   dist/docs/assets/highlight.css
npm notice 35.2kB dist/docs/assets/icons.css
npm notice 9.6kB  dist/docs/assets/icons.png
npm notice 28.1kB dist/docs/assets/icons@2x.png
npm notice 43.1kB dist/docs/assets/main.js
npm notice 2.7kB  dist/docs/assets/search.js
npm notice 32.5kB dist/docs/assets/style.css
npm notice 480B   dist/docs/assets/widgets.png
npm notice 855B   dist/docs/assets/widgets@2x.png
npm notice 4.7kB  dist/docs/index.html
npm notice 3.3kB  dist/docs/modules.html
npm notice 5.7kB  dist/docs/modules/core.html
npm notice 5.8kB  dist/docs/modules/index.html
npm notice 3.8kB  dist/docs/typedoc.json
npm notice 43.8kB dist/index.js
npm notice 592B   dist/types/core/errors.d.ts
npm notice 11.8kB dist/types/core/factories.d.ts
npm notice 862B   dist/types/core/index.d.ts
npm notice 5.3kB  dist/types/core/validation.d.ts
npm notice 515B   dist/types/index.d.ts
npm notice 564B   dist/types/utilities/types.d.ts
npm notice 985B   dist/types/utilities/validation.d.ts
npm notice 2.7kB  package.json
```

Reply:
https://npm.github.io/publishing-pkgs-docs/publishing/the-npmignore-file.html

Reply:
Added a .npmignore to ignore the unnecessary files